.expeditor/verify_linux_pipeline.yml.template

```diff
@@ -0,0 +1,83 @@
+expeditor:
+  defaults:
+    buildkite:
+      timeout_in_minutes: 60
+
+steps:
+  - label: "[@@plan@@] :linux: :habitat: Check for bad patterns"
```

Comment:
These checks are awesome - we should document either the steps to run the checks at your workstation before opening a PR; or indicate that it's OK to open a PR and let buildkite do the checking for you.